README.md

```diff
@@ -13,21 +13,33 @@
 [![][license-epl img]][license-epl]
 [![][license-edl img]][license-edl]
 
-![logo](artwork/eclipse-collections-logo.png) 
+<a href="http://www.eclipse.org/collections"><img src="https://github.com/eclipse/eclipse-collections/blob/master/artwork/eclipse-collections-logo.png" height="50%" width="50%"></a>
 
+#### [中文](https://www.eclipse.org/collections/cn/index.html) | [Deutsch](https://www.eclipse.org/collections/de/index.html) | [Français](https://www.eclipse.org/collections/fr/index.html) | [日本語](https://www.eclipse.org/collections/ja/index.html) | [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html) | [Русский](https://www.eclipse.org/collections/ru/index.html)
+Eclipse Collections is a comprehensive collections library for Java. The library enables productivity and performance by delivering an eloquent, elegant and efficient set of APIs and types. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
 
-## [Eclipse Collections](http://www.eclipse.org/collections)
-##### [中文](https://www.eclipse.org/collections/cn/index.html) | [Deutsch](https://www.eclipse.org/collections/de/index.html) | [Français](https://www.eclipse.org/collections/fr/index.html) | [日本語](https://www.eclipse.org/collections/ja/index.html) | [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html) | [Русский](https://www.eclipse.org/collections/ru/index.html)
-Eclipse Collections is a collections library for Java with a rich, functional and fluent API. The library provides optimized List, Set and Map implementations and has data structures not found in the JDK including Bags, Multimaps and BiMaps. There is support for primitive versions of Lists, Sets, Bags, Stacks and Maps for all primitive types. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
+
+## Why Eclipse Collections?
+
+* Productivity
+    * Rich, functional, and fluent APIs with great symmetry 
+    * List, Set, Bag, Stack, Map, Multimap, BiMap, Interval Types 
+    * Readable, Mutable, and Immutable Types
+    * Mutable and Immutable Collection Factories
+    * Adapters and Utility classes for JCF Types
+* Performance
+    * Memory Efficient Containers 
+    * Optimized Eager, Lazy and Parallel APIs
+    * Primitive Collections for all primitive types 
+* Actively developed since 2005
```

Comment:
If you want to add links to articles later, this would be a good place.

Reply:
I'll link to articles and blogs in the Learning section